tests/integration/simple_integration_test.go

```diff
@@ -0,0 +1,164 @@
+package integration_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"io/ioutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+var (
+	namespace = "k8ssandra"
+	releaseName string
+	releaseNameCluster string
+)
+
+var _ = Describe("Install the cluster", func() {
+	// TODO This assumes the kind was started with correctly setup cluster. Could we automate it here to provider proper e2e?
```

Comment:
I just want to point out that while kind is often used during development, we want to maintain the freedom to run against other types of clusters.

Reply:
There's nothing really expecting kind, other than "a cluster must be up" as this test does not spin one.